libs/langgraph/src/prebuilt/react_agent_executor.ts

```diff
@@ -160,9 +150,7 @@ export function createReactAgent(
     return { messages: [await modelRunnable.invoke(messages, config)] };
   };
 
-  const workflow = new StateGraph<AgentState>({
-    channels: schema,
-  })
+  const workflow = new StateGraph(MessagesAnnotation)
```

Comment:
This is done primarily to get the correct input types (based off the `messagesReducer`, but open to different solutions.

Reply:
I think it's good to make this change, the other syntax is deprecated